config/config.yaml

```diff
@@ -25,12 +25,17 @@ RPM: 10
 
 #### for Search
 
+## Supported values: serpapi/google/serper/ddg
+#SEARCH_ENGINE: serpapi
```

Comment:
Run with error:
```
pydantic.error_wrappers.ValidationError: 1 validation error for SerpAPIWrapper
serpapi_api_key
  To use, make sure you provide the serpapi_api_key when constructing an object. Alternatively, ensure that the environment variable SERPAPI_API_KEY is set with your API key. You can obtain an API key from https://serpapi.com/. (type=value_error)
```
We should allow users to leave this value blank.

Reply:
The issue arises due to the absence of a set SERPAPI_API_KEY, which is required when using the SerpAPI search engine.